build_runner/test/daemon/daemon_test.dart

```diff
@@ -66,9 +67,11 @@ main() {
     await runPub('a', 'run', args: ['build_runner', 'clean']);
   });
 
-  Future<BuildDaemonClient> _startClient({BuildMode buildMode}) {
+  Future<BuildDaemonClient> _startClient(
+      {BuildMode buildMode, List<String> options}) {
+    options ??= [];
     buildMode ??= BuildMode.Auto;
-    var args = ['run', 'build_runner', 'daemon'];
+    var args = ['run', 'build_runner', 'daemon', ...options];
```

Comment:
This would have the downside it seems of causing options skew if some other build daemon client wants to run as well, but I guess that would be by design in this case?

Reply:
Yes.